turbo.json

```diff
@@ -3,6 +3,7 @@
   "globalDotEnv": [
     ".env"
   ],
+  "globalEnv": ["VITE_MOCKED_SIGNER_PROVIDER_URL"],
```

Comment:
This is not relevant for other packages except `react-dapp`
If we can specify this only for this example package - would be great

Reply:
fixed